bnd.bnd

```diff
@@ -0,0 +1,3 @@
+Import-Package: !java.lang, !java.util.*, *
```

Comment:
Please keep the Java package imports  ( see my comment on https://github.com/apache/sling-org-apache-sling-serviceuser-webconsole/pull/1 )